src/frontend/src/components/NavBar/components/AppNavBar/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { useTreatments } from "@splitsoftware/splitio-react";
+import { Icon } from "czifui";
+import Link from "next/link";
+import React, { MouseEventHandler, useState } from "react";
+import { useUserInfo } from "src/common/queries/auth";
+import { ROUTES } from "src/common/routes";
+import { FEATURE_FLAGS, isFlagOn } from "src/components/Split";
+import { InviteModal } from "src/views/GroupMembersPage/components/MembersTab/components/InviteModal";
+import RightNav from "../RightNav";
+import { GroupDetailsDropdown } from "./components/GroupDetailsDropdown";
+import {
+  DropdownClickTarget,
+  LeftNav,
+  Logo,
+  LogoAnchor,
+  NavBar,
+  NavOrg,
+  Separator,
+  StyledIcon,
+} from "./style";
+
+const AppNavBar = (): JSX.Element => {
```

Comment:
Nit: I'd like a brief comment to document the difference in purpose between `AppNavBar` and `StaticPageNavBar`. We all know what's going on, but just some sort of breadcrumbs for anyone new to the project to explain that the app has "inside, authed only" pages and "outside, public" pages with different nav bars.